draft-ietf-quic-multipath.md

```diff
@@ -1024,6 +1042,56 @@ field. If the Identifier Type if 0x02, the PATH_ABANDON frame MUST only
 be sent on the path that is intended to be abandoned.
 
 
+## PATH_STATUS frame {#path-status-frame}
+
+PATH_STATUS Frame are used by endpoints to inform the peer of the current 
+status of one path, and the peer should send packets according to 
+the preference expressed in these frames. 
+PATH_STATUS frames are formatted as shown in {{fig-path-status-format}}.
+
+~~~
+  PATH_STATUS Frame {
+    Type (i) = TBD-03 (experiments use 0xbaba06),
+    Path Identifier (..),
+    Path Status sequence number (i),
+    Path Status (i),
+  }
+~~~
+{: #fig-path-status-format title="PATH_STATUS Frame Format"}
+
+PATH_STATUS Frames contain the following fields:
+
+Path Identifier: An identifier of the path, which is formatted 
+  as shown in {{fig-path-identifier-format}}. Exactly the same as 
+  the definition of Path Identifier in {#path-abandon-frame}.
+
+Path Status sequence number: A variable-length integer specifying 
+  the sequence number assigned for this PATH_STATUS frame.
+  There is a different path status sequence number space for each path.
```

Comment:
Actually I don't think we need to maintain the sequence number per path on the sender side. As long as the sequence number is larger than the last received for that path, you should apply it.

Reply:
It's ok that we maintain only one monotonically increasing sequence number for all the Path Status frame under the same connection.

Reply:
Updated the definition of sequence number.